sdk-tests/src/test/java/io/dapr/it/pubsub/http/SubscriberService.java

```diff
@@ -12,7 +12,7 @@
 /**
  * Service for subscriber.
  */
-@SpringBootApplication(scanBasePackages = {"io.dapr.springboot", "io.dapr.it.pubsub.http"})
```

Comment:
Why is `"io.dapr.it.pubsub.http"` no longer needed?

Reply:
The package where the application is declared is included by default. The springboot path is added automatically now with this change.